GVFS/GVFS.UnitTests/Virtualization/Projection/SortedFolderEntriesTests.cs

```diff
@@ -98,7 +98,7 @@ public void AddItemAtTheBeginning()
         {
             SortedFolderEntries sfe = SetupDefaultEntries();
             LazyUTF8String name = ConstructLazyUTF8String("((shouldbeatthestart");
-            sfe.AddFolder(name);
+            sfe.AddFile(name, new byte[20]);

```

Comment:
Why was this test (and `AddItemAtEnd`) changed to insert a file rather than a folder?

Reply:
Simply because it is an easier call.  The `AddFolder` method was removed and now there is
```
public FolderData GetOrAddFolder(
    LazyUTF8String[] pathParts,
    int partIndex,
    bool parentIsIncluded,
    SparseFolderData rootSparseFolderData)
```

Which is more setup work to call and checking the order shouldn't matter whether it is a file or a folder

Reply:
Ok thanks!